lab-michael/data/one.txt

```diff
@@ -0,0 +1 @@
+5a6f6d6269652069
```

Comment:
While this is totally valid, you might also try pasting in a bunch of Ipsum text to each file rather than hex text. Try different amounts in each file.